docs/usage.md

```diff
@@ -392,16 +392,46 @@ the code integration is very simple: a single call to `bristlemouth_init`.
 
 ```C
 #include "bristlemouth.h"
+#include "l2.h"
 
 void network_device_power_callback(bool on) {
     // Provide or cut off power to the network chip.
     // On the mote, we set pin PH1 high or low.
 }
 
+static void network_device_interrupt(void) {
+    // Necessary logic that occurs when an interrupt from the 10Base-T1L driver
+    // is called should be placed here (application specific), the below
+    // function call is a must in order to inform L2 that an event must be
+    // handled
+    bm_l2_handle_device_interrupt();
+}
+
 int main(void) {
+    // User specific interrupt register code
+    register_device_interrupt(network_device_interrupt);
+
+    // Information about the device must be configured
+    DeviceCfg device = {
```

Comment:
Are you planning to introduce a change to allow bm_core to set a bristlemouth version? If so, in this PR or a different PR?

Reply:
I believe a different PR.
There is going to be some internal discussion to determine if we want to add that to the info message or create a new message in itself.
If we make a new message it will be a new PR.
Depending on the direction we end up going, it may be best to leave this PR open until we discuss further.